src/components/oculus-touch-controls.js

```diff
@@ -50,8 +51,7 @@ module.exports.Component = registerComponent('oculus-touch-controls', {
       button5: 'surface'
     },
     'right': {
-      axis0: 'thumbstick',
-      axis1: 'thumbstick',
+      axes: {'thumbstick': [0, 1]},
```

Comment:
makes processing harder e.g. you can't just do buttons.forEach ... i would argue we should change buttons to array for consistency instead.